tmeister/static/src/toggles/create/create-env/create-env-modal.component.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import styles from './create-env-modal.css';
+import { useCreateEnv } from './create-env.hooks.js'
+
+export default function CreateEnvModal (props) {
+  const { hide, refetchToggles } = props
+  const [ newEnv, setNewEnv ] = useState('')
+  const [ createName, setCreateName ] = useState('')
+  const [ saving, saveCompleted ] = useCreateEnv(createName)
+  if (saveCompleted) {
+    refetchToggles()
+    hide()
+  }
+  return (
+    <div className="cps-modal">
+      <div className="cps-modal__screen"></div>
+      <div className="cps-modal__dialog cps-card">
+        <div className="cps-card__header cps-subheader-sm">
+          <span>
+            Create a new Environment
+          </span>
+          <a
+            className="cps-modal__dialog__close cps-icon cps-icon-close"
+            onClick={hide}
+          />
```

Comment:
opportunity to use `CprIcon` or `CprButtonIcon` from the right reverend styleguide

Reply:
It would be, but we've ripped out the canopy-styleguide from this project, because we kept breaking it with styleguide changes and this being completely open source it didn't make sense to depend on a dynamically updating 3rd party library.